### PR TITLE
Align cost_to_empty with longTermFeeRate

### PR DESCRIPTION
**Pull request by Xekyo**
This changes the calculation of cost_to_empty to the same sign as other
fees so that they add rather than subtract and increases the feerate for
cost_to_empty to the current longTermFeeRate

### CODE REVIEW

**Comment by Xekyo** [comment]
Should I also increment
```commit = repo.commit("HEAD~9")```?

**Comment by achow101** [comment]
> Should I also increment `commit = repo.commit("HEAD~9")`?

Yes. That commit needs to be the base that the simulation runs on.

File: test/functional/sim_coinselect.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def log_sim_results(self, res_file):
 
         # Remaining utxos and fee stats
         remaining_utxos = self.tester.listunspent()
-        cost_to_empty = Decimal(-1) * Decimal(len(remaining_utxos)) * Decimal(68) * Decimal(0.00001) / Decimal(1000)
+        cost_to_empty = Decimal(1) * Decimal(len(remaining_utxos)) * Decimal(68) * Decimal(0.0001) / Decimal(1000)
```
Comment (achow101) [inline test/functional/sim_coinselect.py:114]: You could just remove `Decimal(1)`. Multiplying by 1 is unnecessary.

Comment (Xekyo) [reply test/functional/sim_coinselect.py:114]: Okay, I wasn't sure whether it was needed here, given that there were other multiplications with Decimal(1) on other lines surrounding this. Will remove.

**Comment by achow101** [comment]
Cherry-picked the commits.

**Pull request closed without merging**